lib/common/options.js

```diff
@@ -223,6 +223,7 @@ Options.defaults = {
     concurrency: 500,
     interval: 0,
     subFolders: true,
+    fallback: '200.html',
```

Comment:
Why not defaulting to 404.html?

Reply:
It's to keep current behaviour default, so no BC breakage occurs.

Reply:
See https://github.com/nuxt/nuxt.js/issues/2690#issuecomment-360879305

Reply:
Okay :)